test/sqllogictest/pg_get_constraintdef.slt

```diff
@@ -0,0 +1,29 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# This file is derived from the logic test suite in CockroachDB. The
+# original file was retrieved on June 10, 2019 from:
+#
+# The original source code is subject to the terms of the Apache
+# 2.0 license, a copy of which can be found in the LICENSE file at the
+# root of this repository.
+
+query T
+SELECT pg_get_constraintdef(pg_constraint.oid) FROM pg_constraint
+----
+
+query error
```

Comment:
This should have more text after error to match the error string on (it's a regex, and can match a part of the error text).